pkg/validation/internal/operatorhub.go

```diff
@@ -21,9 +21,51 @@ import (
 // OperatorHubValidator validates the bundle manifests against the required criteria to publish
 // the projects on OperatorHub.io.
 //
+// This validator will ensure that:
+// - The annotations capabilities into the CSV has a valid option, which are:
+// * "Basic Install"
+// * "Seamless Upgrades"
+// * "Full Lifecycle"
+// * "Deep Insights"
+// * "Auto Pilot"
+// - The annotations categories into the CSV has a valid option, which are:
```

Comment:
we support custom categories too though. maybe worth mentioning.
https://olm.operatorframework.io/docs/tasks/creating-operator-bundle/#custom-bundle-categories